oneview_redfish_toolkit/mockups/oneview/ServerHardwaresProfiledApplied.json

```diff
@@ -0,0 +1,1165 @@
+[{"processorSpeedMhz":2400,
```

Comment:
Consider renaming the file to ServersHardwareProfiled.json or something like that.

Reply:
Can we use ServerProfiles.json or ServerProfilesApplied.json? To follow Oneview naming structure. Let's discuss tomorrow.